README.md

```diff
@@ -1,44 +1,154 @@
 [![Build Status](https://travis-ci.org/IBM/watson-review-analyzer.svg?branch=master)](https://travis-ci.org/IBM/watson-review-analyzer)
 
-# watson-review-analyzer
-An application that ingests unstructured reviews and uses Watson Discovery to find insights and provide a second opinion on the reviews. 
+# Create a Review Analyzer with Watson Discovery
 
-## 1. Create IBM Cloud services
+In this Code Pattern, we will create a Node.js app that takes the reviews from an online shopping website, Amazon, and feeds them into the Watson Discovery service. The reviews will be stored in a Cloudant or couchdb database. The Watson Discovery service will show the overall sentiments of the reviews. The sample application will do all the reading of reviews for you and will give an overall insight about them. The Code Pattern can be useful to developers that are looking into processing multiple documents with Watson Discovery.
 
-Create the following services:
+When the reader has completed this Code Pattern, they will understand how to:
+
+* Interact with Watson Discovery using Watson's Node SDK
+* Build a User Inerface around the result of Watson Discovery
+* Deploy the app in Kubernetes
+* Deploy and connect a CouchDB in the same instance of Kubernetes Cluster
+
+<!--Remember to dump an image in this path-->
+![Architecture](/docs/app-architecture.png)
+
+## Flow
+1. The user deploys the app in Kubernetes. The user interacts with the User Interface of the app.
+2. The user enters the product ID and the app would start getting the reviews of the Product.
+3. The app then stores the reviews in a CouchDB database for later use.
+4. The app starts to upload the reviews in Watson Discovery.
+5. After Watson Discovery finishes processing the reviews, the app then stores the result (General Sentiment and Top Entities) in the CouchDB. The user will see the result in the UI.
```

Comment:
how about:

"When Watson Discovery has finished processing product reviews the app stores the results (sentiment and top entities) in a database. The user is able to see these results in the user interface."